packages/suite/src/support/suite/ThemeProvider.tsx

```diff
@@ -52,13 +54,13 @@ const ThemeProvider: React.FC = ({ children }) => {
         }
 
         // set active theme OS based theme only if there are no saved settings
-        if (storedTheme === null) {
+        if (storedTheme === null && !error) {
```

Comment:
do I understand correctly that you don't want this condition evaluate to `true` if there is error set on line 48? Shouldn't there be `return` after `setError` then?

Reply:
Yup, you do get that right. I don't see how the return would help though, setError is in separate async function and I don't work with its return value.